js/pages/samples.js

```diff
@@ -588,11 +614,22 @@ const renderLocations = (site) => {
                         </div>
                     </div>
                 </div>`
-            } else {
+            } else if(site==u_chicago && !location[1]){
```

Comment:
```suggestion
            } else if (site === u_chicago && !location[1]){
```
Format suggestion and strict equality operator